scripts/us_census/pep/annual_population/preprocess.py

```diff
@@ -0,0 +1,859 @@
+                       USA, USA_GEO_ID, DISTRICT_OF_COLUMBIA_STATE_CODE,
+                       DISTRICT_OF_COLUMBIA_COUNTY_CODE, INPUT_DIR)
+
+sys.path.insert(1, os.path.join(_MODULE_DIR, '../../../../'))
+import util.alpha2_to_dcid as alpha2todcid
+import util.name_to_alpha2 as statetoshortform
+import util.county_to_dcid as countytodcid
+# pylint: enable=wrong-import-position
+# pylint: enable=import-error
+
+USSTATE_MAP = alpha2todcid.USSTATE_MAP
+COUNTY_MAP = countytodcid.COUNTY_MAP
+_USSTATE_SHORT_FORM = statetoshortform.USSTATE_MAP
+
+_STATE_CONFIG_PATH = os.path.join(_MODULE_DIR, "states_config.json")
+with open(_STATE_CONFIG_PATH, encoding="utf-8") as states_file:
+    _STATE_CONFIG = json.load(states_file)
+
+_MCF_TEMPLATE = ("Node: dcid:Count_Person\n"
+                 "typeOf: dcs:StatisticalVariable\n"
+                 "populationType: dcs:Person\n"
+                 "statType: dcs:measuredValue\n"
+                 "measuredProperty: dcs:count\n")
+
+_TMCF_TEMPLATE = ("Node: E:USA_Annual_Population->E0\n"
+                  "typeOf: dcs:StatVarObservation\n"
+                  "variableMeasured: dcs:Count_Person\n"
+                  "measurementMethod: dcs:CensusPEPSurvey\n"
+                  "observationAbout: C:USA_Annual_Population->Location\n"
+                  "observationDate: C:USA_Annual_Population->Year\n"
+                  "observationPeriod: \"P1Y\"\n"
+                  "value: C:USA_Annual_Population->Count_Person \n")
+
+
+def _load_data_df(path: str,
+                  file_format: str,
+                  header: str = None,
+                  skip_rows: int = None,
+                  encoding: str = None) -> pd.DataFrame:
+    """
+    Returns the DataFrame using input path and config.
+    Args:
+        path (str): Input File Path
+        file_format (str): Input File Format
+        header (str, optional): Input File Header Index. Defaults to None.
+        skip_rows (int, optional): Skip Rows Value. Defaults to None.
+        encoding (str, optional): Input File Encoding. Defaults to None.
+
+    Returns:
+        data_df (pd.DataFrame): Dataframe of input file
+    """
+    data_df = None
+    if file_format.lower().endswith("csv"):
+        data_df = pd.read_csv(path, header=header, encoding=encoding)
+    elif file_format.lower() == "txt":
+        data_df = pd.read_table(path,
+                                index_col=False,
+                                delim_whitespace=True,
+                                engine='python',
+                                header=header,
+                                skiprows=skip_rows)
+    elif file_format.lower() in ["xls", "xlsx"]:
+        data_df = pd.read_excel(path, header=header)
+    return data_df
+
+
+def _geo_id(val: str) -> str:
+    """
+    Returns GeoID for State Or County.
+    Args:
+        val (str): FIPS Code
+
+    Returns:
+        str: State/County GeoId
+    """
+    res = "geoId/"
+    state, county = val[0], val[1]
```

Comment:
is `val` a string or a list? picking first char as state  and second char as county seems incorrect.

Reply:
It is actually pandas series. Updated the function definition

Reply:
The function definition still says `val:str`. Am i missing a refresh or recent code change?